docker-compose.yml

```
@@ -31,18 +31,19 @@ services:
       - db
     # Configura as variáveis de ambiente dentro do container
     environment:
-      MYSQL_USER: root
-      MYSQL_PASSWORD: password
-      HOSTNAME: db
-      MYSQL_PORT: 3306
+      DB_USER: 'root'
+      DB_PASSWORD: 'password'
```

Só mantendo a mesma senha do dia 1, ok?

```suggestion
      DB_PASSWORD: 'password123'
```